packages/core/src/tracing/supportedVersion.js

```diff
@@ -9,5 +9,5 @@ const semver = require('semver');
 
 /** @type {(version: string) => boolean} */
 module.exports = exports = function supportedVersion(version) {
-  return semver.satisfies(version, '>=14.0.0');
+  return semver.satisfies(version, '>=14.0.0', { includePrerelease: true });
```

Comment:
issue: Please add technical context how this works. 

(This affects the production code. It would allow customers to use any prerelease. This needs to be discussed/agreed first.)

Reply:
> This affects the production code

@aryamohanan Whats your opinion on that?

Reply:
IMO: Including prerelease versions in production may not be ideal. We could implement a flag specifically for test environments that activates the includePrerelease option. This approach would keep the production code stable by allowing only tested, stable versions, while enabling prerelease versions for test environments.
```suggestion
  const includePrerelease = process.env.NODE_ENV === 'test';
  return semver.satisfies(version, '>=14.0.0', { includePrerelease });
```

Reply:
That would be great!